src/Underscore.php

```diff
@@ -83,9 +119,13 @@ public function __call($method, $args)
      */
     public static function __callStatic($method, $args)
     {
-        $payloadClass = sprintf('\Underscore\Initializer\%sInitializer', ucfirst($method));
-        /** @var $payload callable */
-        $payload = new $payloadClass();
+        $payload = static::fromMixin($method);
```

Comment:
Should that be done via flag for the `mixin` method? For example:

```php
Underscore::mixin($functions, Underscore::MUTATOR);
Underscore::mixin($functions, Underscore::ACCESSOR);
Underscore::mixin($functions, Underscore::INITIALIZER);
```

Honestly, I wasn't really that concerned with differentiating between types for mixins, because all method names are unique... you couldn't have an initializer called `foo` and a mutator called `foo`, the interface doesn't allow it.